tests/conftest.py

```diff
@@ -304,7 +307,8 @@ def _navigate_function(text=None, use_clipboard=False):
             EC.any_of(
                 EC.presence_of_element_located((By.CSS_SELECTOR, ".loaded")),
                 EC.title_contains(text),
-                EC.url_contains("localhost"),  # MozSearch/test website server
+                EC.url_contains("localhost"),  # test website server
+                EC.url_contains("static-server"),  # MozSearch server
```

Comment:
Wait, maybe I'm misreading this&hellip;
Before I thought it was just checking that there was a "localhost" in https://localhost:port/
How does the URL contain _both_ "localhost" and "static-server"? Is static-server a pathname?

Reply:
It's an either or, so it waits until any of these are true

Reply:
Ah, sorry, missed that `any_of()`. 👍